packages/runtime/datastore-definitions/src/dataStoreRuntime.ts

```diff
@@ -34,6 +34,7 @@ export interface IFluidDataStoreRuntimeEvents extends IEvent {
 	(event: "op", listener: (message: ISequencedDocumentMessage) => void);
 	(event: "signal", listener: (message: IInboundSignalMessage, local: boolean) => void);
 	(event: "connected", listener: (clientId: string) => void);
+	(event: "readonly", listener: (isReadOnly: boolean) => void);
```

Comment:
Docs? Ideally, all of these events would be documented, but we should at least ensure that new ones we add are.

Reply:
Same with `isReadOnly`

Reply:
```suggestion
	/* 
	 * The readonly event is fired when the readonly state of the datastore runtime changes.
	 * The isReadOnly will express the new readonly state.
	 */
	(event: "readonly", listener: (isReadOnly: boolean) => void);
```